doc/config.toml

```diff
@@ -0,0 +1,33 @@
+baseURL = "http://localhost"
+languageCode = "en-us"
+title = "The Things Network Stack for LoRaWAN"
+theme = "hugo-theme-techdoc"
+
+enableGitInfo = true
+
+[frontmatter]
+  date = [":git", "date", "publishDate", "lastmod"]
+  lastmod = [":git", "lastmod", "date", "publishDate"]
+  publishDate = ["publishDate", "date"]
+  expiryDate = ["expiryDate"]
+
+[Params]
+  # Souce Code repository section
+  description = "The Things Network Stack for LoRaWAN is an open-source LoRaWAN network stack suitable for large, global and geo-distributed public and private networks as well as smaller networks."
+  github_repository = "https://github.com/TheThingsNetwork/lorawan-stack"
+  version = "3.0.0"
```

Comment:
This should be filled out, also through `mage` versioning commands, right @htdvisser ?